server/util/redact/redact.go

```diff
@@ -187,9 +189,18 @@ func stripRemoteHeadersFromCmdLine(tokens []string) {
 	}
 }
 
+func stripExplicitCommandLineFromCmdLine(tokens []string) {
+	for i, token := range tokens {
+		if strings.HasPrefix(token, buildMetadataOptionPrefix+explicitCommandLineName+"=") {
+			tokens[i] = ""
```

Comment:
clearing out the arg (instead of dropping) might cause issues later, since a literal `""` argument is treated by bazel as an error, but seems fine for now since we won't actually render the `""` in the UI. just something to be aware of in case we see errors in the future

```
ERROR: Skipping '': invalid target name '': empty target name
```

Reply:
Ack, this should just be for UI rendering - we can switch to filtering if it causes issues.